pkg/trace/obfuscate/sql_test.go

```diff
@@ -545,6 +545,21 @@ func TestSQLTableFinderAndReplaceDigits(t *testing.T) {
 				tables:     "test",
 				obfuscated: "select * from test where ! - ?",
 			},
+			{
+				query:      "select count(*) as `count(*)` from test",
```

Comment:
Can you please remove the keys from these structs? Including the ones you've added in your last PR. Just to stay consistent. Or, if you feel strongly about having keys, then please add them everywhere in the slice.

Reply:
I do prefer when keys are defined but I actually didn't do it on purpose here (I was using IntelliJ's "fill in all keys" which did this) and this isn't a case I feel strongly about. 

Updated ✅